internal/hns/hnsendpoint.go

```diff
@@ -94,6 +94,27 @@ func GetHNSEndpointByName(endpointName string) (*HNSEndpoint, error) {
 	return nil, EndpointNotFoundError{EndpointName: endpointName}
 }
 
+type endpointAttachInfo struct {
+	SharedContainers json.RawMessage `json:",omitempty"`
+}
+
+func (endpoint *HNSEndpoint) IsAttached(vID string) (bool, error) {
+	attachInfo := endpointAttachInfo{}
+	err := hnsCall("GET", "/endpoints/"+endpoint.Id, "", &attachInfo)
+
+	// Return true allows us to just return the err if this is true
+	if err != nil {
+		return true, err
```

Comment:
Seems like an odd pattern. Why not `false, err`. If a person doesnt check `err` they would assume from `true` that it is there when in fact they dont know.

Reply:
done